README.md

```diff
@@ -48,27 +48,41 @@ WakaTime gives you an idea of the time you really spent on coding. This helps yo
 
 > You wouldn't need an GitHub Access Token since GitHub Actions already makes one for you.
 
-Here is a sample workflow file for you to get started:
+Please follow the steps below:
+
+1. Go to your `<username>/<username>/actions`, hit `New workflow`, `set up a workflow yourself`, delete all the default content github made for you.
+2. Copy the following code and paste it to your new workflow you created at step 1:
+  ```yml
+  name: Waka Readme
+
+  on:
+    schedule:
+      # Runs at 12am UTC
+      - cron: '0 0 * * *'
```

Comment:
```suggestion
    workflow_dispatch
    schedule:
      # Runs at 12am UTC
      - cron: '0 0 * * *'
```

This will resolve the issue of manually triggering the workflow